hack/common.sh

```diff
@@ -41,7 +41,7 @@ GINKGO_TEST_SUITE="${GINKGO_TEST_SUITE:-ocs}"
 # This env var allows developers to point to a custom oc tool that isn't in $PATH
 # defaults to just using the 'oc' binary provided in $PATH
 OCS_OC_PATH="${OCS_OC_PATH:-oc}"
-OCS_FINAL_DIR="deploy/bundle/manifests"
+OCS_FINAL_DIR="deploy/ocs-operator/manifests"
```

Comment:
If we have decided to move the directories, Lets stop using the deploy directory. Let's have a directory called bundle at the root of the repo. And store bundle-related stuff inside that.

OCS_FINAL_DIR="bundle/ocs-operator/manifests"

Reply:
@iamniting WDYT?

Reply:
Moving things around is not in the scope of this PR. It's just allowing a new bundle to co-exist.